ipapython/ipa_log_manager.py

```diff
@@ -54,6 +54,20 @@
 # Used by standard_logging_setup() for file message
 LOGGING_FORMAT_STANDARD_FILE = '%(asctime)s %(levelname)s %(message)s'
 
+# Used by standard_logging_setup() for command line output
+LOGGING_FORMAT_INFO_CMD = '%(message)s'
+
+
+# Additional info_log level methods
+def log_info_cmd(obj, msg, *args, **kwargs):
+    kwargs.setdefault("extra", {})["cmd_output"] = True
+    obj._log(logging.INFO, msg, args, **kwargs)
+
+
+def log_info_cmd_root(msg, *args, **kwargs):
```

Comment:
We shouldn't use the root logger at all. If some module uses the root logger, then please fix the module instead.

Reply:
This is used any more with the latest version.